docs/contracts/accounting-oracle.md

```diff
@@ -3,14 +3,18 @@
 - [Source code](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/oracle/AccountingOracle.sol)
 - [Deployed contract](https://etherscan.io/address/0x852deD011285fe67063a08005c71a85690503Cee)
 
+:::note
+Some methods, events and exceptions are inherited from the [BaseOracle](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/oracle/BaseOracle.sol) contract
+:::
+
 AccountingOracle is a contract where oracles send addresses' balances controlled by the DAO on the Consensus Layer side.
```

Comment:
Not only balances, but a lot of other things, that also important. Can suggest to rephrase in more broad manner, because we have a detailed view in `ReportData` section.

Reply:
yep